course7/homeworks/mart/complex/complex.cc

```diff
@@ -0,0 +1,182 @@
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <stack>
+#include <regex>
+
+struct Complex
+{
+	Complex() = default;
+	Complex(double r) : real(r) {}
+	Complex(double r, double i) : real(r) , imaginary(i) {}
+	~Complex() = default;
```

Comment:
This is not required. destructor is always generated.
The purpose of ```= default``` is to ask compiler to generate the method, if otherwise it would refrained for it.

Reply:
Thx, fixed